reviews/utils/review_utils.py

```diff
@@ -264,6 +268,53 @@ def remind_reviewers():
         )
 
 
+def remind_supervisor_reviewers():
+    """
+    Sends an email to a supervisor reviewer to remind them to review a proposal.
+    The reminders are only sent for open reviews older than a week.
+    """
+
+    today = datetime.date.today()
+
+    decisions = Decision.objects.filter(
+        review__is_committee_review=False,
+        review__stage=Review.Stages.SUPERVISOR,
+        review__date_should_end__lte=today,
+    )
+
+    for decision in decisions:
+        proposal = decision.review.proposal
+        subject = "Herinnering: beoordeel aanvraag {}".format(
+            proposal.committee_prefixed_refnum(),
+        )
+        params = {
+            "supervisor": decision.reviewer.get_full_name(),
+            "creator": proposal.created_by.get_full_name(),
+            "proposal_url": settings.BASE_URL
+            + reverse("reviews:decide", args=(decision.pk,)),
```

Comment:
So this var isn't used in the template, as Desiree's text didn't contain it. It's here because it was in the other reminder mail code I copied, so I left it in for the inevitable 'why isn't there a link' comments. 

Feel free to nudge me to add it to the e-mail, but I'm tired atm :P